packages/php-datatypes/src/Definition/Exasol.php

```diff
@@ -1,446 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Keboola\Datatype\Definition;
-
-use Keboola\Datatype\Definition\Exception\InvalidLengthException;
-use Keboola\Datatype\Definition\Exception\InvalidOptionException;
-use Keboola\Datatype\Definition\Exception\InvalidTypeException;
-use LogicException;
-
-/**
- * Class Exasol
- *
- * https://docs.exasol.com/sql_references/data_types/datatypedetails.htm
- */
-class Exasol extends Common
```

Comment:
jses si jistej, ze to nikde nikho nepouziva? nejakej extractor nebo neco. vetsinou, jestli je nekde neco pouzite z nasich libek, tak je to datatypes